code/parse/sexp.h

```diff
@@ -746,6 +746,8 @@ class waypoint_list;
 #define OP_MODIFY_VARIABLE_XSTR				(0X0030 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// m!m
 #define OP_RESET_POST_EFFECTS				(0x0031 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// Goober5000
 #define OP_ADD_REMOVE_HOTKEY				(0x0032 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)    // wookieejedi
+#define OP_TECH_REMOVE_INTEL_XSTR			(0x0033 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)    // wookieejedi
+#define OP_TECH_REMOVE_INTEL				(0x0034 | OP_CATEGORY_CHANGE  | OP_NONCAMPAIGN_FLAG)    // wookieejedi
```

Comment:
`CHANGE2` should be used here, because otherwise it's the same ID as another existing sexp.